proto/cosmos/evm/vm/v1/evm.proto

```diff
@@ -20,21 +20,20 @@ message Params {
     (gogoproto.customname) = "ExtraEIPs",
     (gogoproto.moretags) = "yaml:\"extra_eips\""
   ];
+  reserved 5;
+  reserved "chain_config";
   // allow_unprotected_txs defines if replay-protected (i.e non EIP155
   // signed) transactions can be executed on the state machine.
-  bool allow_unprotected_txs = 5;
+  bool allow_unprotected_txs = 6;
   // renamed active_precompiles to active_static_precompiles
-  reserved 6;
+  reserved 7;
   // evm_channels is the list of channel identifiers from EVM compatible chains
-  repeated string evm_channels = 7 [ (gogoproto.customname) = "EVMChannels" ];
+  repeated string evm_channels = 8 [(gogoproto.customname) = "EVMChannels"];
```

Comment:
Aren't these breaking changes an issue?

Reply:
reverted to cosmos evm params proto and added params migration